modules/accredited_representative_portal/app/services/accredited_representative_portal/power_of_attorney_request_service/accept.rb

```diff
@@ -80,6 +87,8 @@ def create_error_form_submission(message, response_body)
           service_response: response_body,
           error_message: message
         )
+
+        StatsD.increment('ar.poa.submission.enqueue_failed.count')
```

Comment:
We don't lose anything by fitting this into the duration distribution that we use for the other terminal POA submission statuses. So we can emit the 2x metrics w/ 0s here:
- `ar.poa.submission.duration`
- `ar.poa.submission.enqueue_failed.duration`

In DD distribution metrics can power all of:
- Histograms
- Timeseries
- Totals

Specifically, in DD continuous value metrics such as a duration can be counted individually as events.